### PR TITLE
Enhancement/services

### PR DESCRIPTION
**Pull request by tomazmm**
* Refactoring some code parts
* Cloning Lachesis exporter git repo and not by go get

### CODE REVIEW

File: resources/run.sh [hunk 1]
```diff
@@ -55,15 +54,14 @@ while sleep 30; do
...
-  echo "Node_exporter: $NODE_EXPORTER_STATUS"
-  echo "Prometheus: $PROMETHEUS_STATUS"
-  echo "Grafana: $GRAFANA_STATUS"
-  echo "Lachesis: $LACHESIS_EXPORTER_STATUS"
-  # If the greps above find anything, they exit with 0 status
-  # If they are not both 0, then something is wrong
-  if [ $NODE_EXPORTER_STATUS -ne 0 -o $LACHESIS_EXPORTER_STATUS -ne 0 ] ||
-     [ $GRAFANA_STATUS -ne 0 -o $PROMETHEUS_STATUS -ne 0 ]; then
-    echo "One of the processes has already exited."
-    exit 1
```
Comment (DollyVolley) [inline resources/run.sh:67]: Was this the reason why it crashed and didn't recover?

**Review by DollyVolley — APPROVED** [review]
Nice build and setup utility!

**Comment by DollyVolley** [comment]
PR looks good! I just saw that we still have an open PR from @mr-rooftop who made the next logical step and used docker-compose. I remember having severe troubles starting it up with compose but we might should take another look as it even provides more UX. 

And please see question above!

File: resources/run.sh (hunk 1 above)
Comment (tomazmm) [reply resources/run.sh:67]: No, the reason was weird bug in lachesis exporter, which only occurred when running in docker and not when executing binary on host.

**Pull request merged**